backend/src/main/java/com/woowacourse/momo/group/service/specification/GroupAttribute.java

```diff
@@ -0,0 +1,37 @@
+package com.woowacourse.momo.group.service.specification;
+
+import java.util.List;
+
+import javax.persistence.criteria.Expression;
+import javax.persistence.criteria.Path;
+import javax.persistence.criteria.Root;
+
+import com.woowacourse.momo.group.domain.group.Group;
+
+public enum GroupAttribute {
+
+    ID("id"),
+    HOST("host"),
+    NAME("name", "value"),
+    CATEGORY("category"),
+    CAPACITY("participants", "capacity"),
+    DEADLINE("calendar", "deadline"),
+    DESCRIPTION("description"),
+    IS_EARLY_CLOSED("isEarlyClosed");
+
```

Comment:
```suggestion
    DESCRIPTION("description"),
    IS_EARLY_CLOSED("isEarlyClosed"),
    ;
```
몇몇 ENUM 클래스들은 마지막 값을 추가 후에 `,` 를 넣고 그 후에 `;`를 넣었더라고요! 해당 내용도 통일을 하면 좋을 것 같습니다!